src/amo/components/Footer/index.js

```diff
@@ -143,6 +136,81 @@ export class FooterBase extends React.Component {
                 </a>
               </li>
             </ul>
+          </section>
+
+          <section className="Footer-product-links">
+            <h4 className="Footer-links-header">{i18n.gettext('Products')}</h4>
+            <ul className="Footer-links">
+              <li>
+                <a
+                  className="Footer-lockwise-link"
+                  href={`https://www.mozilla.org/firefox/lockwise/${makeQueryStringWithUTM(
+                    {
+                      utm_source: 'addons.mozilla.org',
+                      utm_medium: 'referral',
+                      utm_content: 'footer-link',
+                      utm_campaign: null,
+                    },
+                  )}`}
+                >
+                  {i18n.gettext('Lockwise')}
```

Comment:
Sorry about the back-and-forth, but I didn't catch this before:  `Products` and `Browsers` should be translated, as it is right now, but `Lockwise`, `Monitor`, `Send` and `Pocket` are product names and should *not* be translated.